### DISCUSSION TITLE
Time adjustment support in Dashboard view

### DISCUSSION THREAD

**Discussion by RoryCrispin** [grafana/clickhouse-datasource#405] [Ideas]
<img width="98" alt="Screenshot 2023-05-30 at 11 13 19" src="https://github.com/grafana/clickhouse-datasource/assets/716269/862c018a-5847-4327-89c8-fad891e8e3f3">

Explore view supports time window adjustment and has 'Newer/Older logs' buttons. These would be great to have in Dashboard view too.

Workaround:
If you have a timeseries chart visible on the dashboard, then it's possible to filter the dashboard time window down by highlighting a region in the chart. There is no way to expand the time window, however.

**Comment by mshustov** [comment]
Not supported by Grafana dashboards.
Could you add a comment to the Grafana issue https://github.com/grafana/grafana/issues/69099 to demonstrate the demand for this functionality? 
The proposed solution is to expand the timerange.
